src/bp/ui-studio/src/web/components/Layout/WarningMessage/style.scss

```diff
@@ -0,0 +1,30 @@
+.warning {
+  position: relative;
+  background: var(--hover-lighthouse-30);
+  padding: var(--spacing-large) var(--spacing-medium);
+  border-radius: 5px;
+  display: flex;
+  align-items: center;
+
+  &_container {
+    position: fixed;
+    right: 50%;
+    bottom: calc(var(--spacing-medium) + 28px);
+    transform: translate(50%, 0);
```

Comment:
are you a wizard? :fearful:

Reply:
![image](https://user-images.githubusercontent.com/3555826/91599302-11893f80-e934-11ea-9e43-d1ad902479c8.png)